src/main/java/wooteco/subway/auth/application/AuthService.java

```diff
@@ -1,7 +1,41 @@
 package wooteco.subway.auth.application;
 
 import org.springframework.stereotype.Service;
+import wooteco.subway.auth.dto.TokenRequest;
+import wooteco.subway.auth.dto.TokenResponse;
+import wooteco.subway.auth.exception.AuthorizationException;
+import wooteco.subway.auth.infrastructure.JwtTokenProvider;
+import wooteco.subway.member.dao.MemberDao;
+import wooteco.subway.member.domain.Member;
 
 @Service
 public class AuthService {
+
+    private MemberDao memberDao;
+    private JwtTokenProvider jwtTokenProvider;
```

Comment:
```suggestion
    private final MemberDao memberDao;
    private final JwtTokenProvider jwtTokenProvider;
```

`final` 키워드를 빼는 작업을 별도로 하신 이유가 있을까요?

Reply:
기존 제공받은 코드가 상수를 제외하고는 final을 붙이지 않아서 컨벤션이라고 생각해서 따랐습니다.
3단계를 진행하며 프로젝트 대상으로 필드에 final을 붙이는 리팩토링을 했습니다.